### ISSUE TITLE
Accessibility: Add U-M Users One Section progress indicator

### ISSUE THREAD

**Issue by ktowneUM** [tl-its-umich-edu/canvas-course-manager-next#635]
Based on the [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

Severity: Critical

Feature: Add U-M Users One Section

Issue notes:
Screen reader did not announce the progress indicator or indicate that is a part of "Select - Upload - Review - Confirmation"

<img width="1048" height="133" alt="Image" src="https://github.com/user-attachments/assets/67a73052-54f7-43e0-8750-d0dd29badc83" />

Criterion:
